services/api/src/admin_new_user.py

```diff
@@ -143,6 +146,23 @@ class AdminNewUserSchema(Schema):
     )
 
 
+def enforce_add_user_org_permissions(
```

Comment:
We've got pretty similary "enforce_xxx_permissions" in several of these admin file additions. Could we abstract to a shared location to reuse code?